src/app/api/document/documentManager.ts

```
@@ -0,0 +1,90 @@
+      }
+
+      const data = await response.json();
+      this.currentDocId = docId;
+      return data;
+    } catch (error) {
+      console.error('Error loading document:', error);
+      throw error;
+    }
+  }
+
+  startAutoSave(state: EditorState, interval: number = 30000): void {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)